services/removeArchived.js

```diff
@@ -0,0 +1,66 @@
+import AWS from 'aws-sdk'; // eslint-disable-line import/no-extraneous-dependencies, import/no-unresolved
+import { difference } from 'lodash';
+
+/* eslint-disable no-console */
+
+const s3 = new AWS.S3({
+  apiVersion: '2006-03-01',
+  region: 'eu-west-1',
+});
+
+const peoplePath = 'about-us/people/';
+const jobsPath = 'about-us/join-us/';
+
+function extractSlugFromKey(key) {
+  const match = key.match(/about-us\/[\w-]+\/([\w-]+)\/index.html/);
+  return match && match[1];
+}
+
+const getLiveBadgers = (Bucket, Prefix) => s3.listObjects({ Bucket, Prefix }).promise();
+const deleteObjects = (bucketName, keysToDelete) =>
+  s3
+    .deleteObjects({
+      Bucket: bucketName,
+      Delete: {
+        Objects: keysToDelete,
+        Quiet: false,
+      },
+    })
+    .promise();
+
+function removeArchivedInPath(bucketName, actualSlugs, s3Prefix) {
```

Comment:
if you prefer path over prefix I guess all i'm saying is perhaps we should be be constant. My preference is to stick with whatever amazon defines them as.